Dockerfile

```diff
@@ -1,3 +1,4 @@
+# renovate: datasource=github-releases depName=microsoft/ApplicationInsights-Java
```

Comment:
This is not necessary.

We don't have a reference to AppInsights in the Dockerfile, no point asking renovate to update it.

This line you need only if in Dockerfile you have something on the line of `ARG APP_INSIGHTS_AGENT_VERSION=x.y.z`

Reply:
Thanks for the Clarification on this Filippo. I have reverted the renovate in this file and updated the PR. Please take a look now.